wagtail/admin/widgets/tags.py

```diff
@@ -31,6 +31,13 @@ def get_context(self, name, value, attrs):
         else:
             free_tagging = self.free_tagging
 
+        tag_spaces_allowed = getattr(settings, "TAG_SPACES_ALLOWED", True)
+        if tag_spaces_allowed:
+            tag_help_text = "Multi-word tags with space will be enclosed in quotes and separated by comma."
```

Comment:
We must always use translations and never use bare strings, otherwise this will not work for translations.

add `from django.utils.translation import gettext_lazy as _` in your imports.

Plus a bit of a reword of this.
```suggestion
            tag_help_text = _("Multi-word tags with spaces will automatically be enclosed in double quotes (\").")
```